### PR TITLE
fix: payment query and og bugs

### PR DESCRIPTION
**Pull request by sruddy**

### CODE REVIEW

File: packages/dapp/src/components/StoryFeed/StoryFeed.tsx [hunk 1]
```diff
@@ -76,20 +76,28 @@ class StoryFeedPage extends React.Component {
...
-                  openGraphData={storyData.openGraphData}
-                  displayedContributors={storyData.payments}
-                  sortedContributors={storyData.payments}
-                  totalContributors={storyData.payments ? storyData.payments.length : 0}
-                />
-              ));
+              return feedQueryData.postsSearch.posts.map((storyData: any, i: number) => {
+                return (
+                  <>
+                    {storyData.openGraphData && storyData.openGraphData.title && storyData.openGraphData.url && (
```
Comment (sruddy) [inline packages/dapp/src/components/StoryFeed/StoryFeed.tsx]: Require at minimum og title and url. This shouldn't be a problem for 99.9% of newsrooms but might come up. We should also let them know somewhere what og data we use.

**Comment by dankins** [comment]
:tada: This PR is included in version 4.8.31 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**